test/integration/acceptance/make_peer_pointee_matcher.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include "interfaces/common_objects/string_view_types.hpp"
+#include "interfaces/common_objects/types.hpp"
+
+auto makePeerPointeeMatcher(
```

Comment:
```suggestion
inline auto makePeerPointeeMatcher(
```

Reply:
Oh, I completely forgot about it :facepalm: